core/src/main/java/io/fabric8/maven/core/access/ClusterAccess.java

```diff
@@ -72,6 +73,18 @@ public String getNamespace() {
         return namespace;
     }
 
+    /**
+     * Returns true if this cluster is a traditional OpenShift cluster with the <code>/oapi</code> REST API
+     * or supports the new <code>/apis/image.openshift.io</code> API Group
+     */
```

Comment:
see https://github.com/fabric8io/kubernetes-client/blob/master/openshift-client/src/main/java/io/fabric8/openshift/client/OpenShiftClient.java#L150-L153 and https://github.com/fabric8io/kubernetes-client/blob/master/openshift-client/src/main/java/io/fabric8/openshift/client/DefaultOpenShiftClient.java#L423